pom.xml

```diff
@@ -69,8 +69,8 @@
     <changelist>-SNAPSHOT</changelist>
     <jenkins.version>2.107.3</jenkins.version>
     <java.level>8</java.level>
-    <scm-api.version>2.3.2</scm-api.version>
-    <git-plugin.version>3.3.0</git-plugin.version>
+    <scm-api.version>2.4.0</scm-api.version>
+    <git-plugin.version>4.0.0-rc</git-plugin.version>
```

Comment:
Not sure that I'm very comfortable with depending on an "rc" version of a plugin that is also removed from the update center?

Reply:
This is for `test` scope.